honeybee_radiance/postprocess/annualdaylight.py

```diff
@@ -54,19 +54,19 @@ def filter_schedule_by_hours(sun_up_hours, schedule=None):
         schedule: A list of 8760 values for the occupancy schedule.
 
     Returns:
-        List - A filtered version of the annual schedule that only includes the
-            sun-up-hours.
+        A tuple with two values.
 
+        occ_pattern -- A filtered version of the annual schedule that only
+            includes the sun-up-hours.
+        
+        total_hours -- A number for the total occupied hours of the schedule.
     """
     schedule = schedule or generate_default_schedule()
     occ_pattern = [schedule[int(h)] for h in sun_up_hours]
+    return occ_pattern, sum(schedule)
```

Comment:
Hi @chriswmackey, thank you for this fix. One comment here about using `sum(schedule)`. From the other lines in the code when you do `if is_occ < 0.5:` it looks like you are allowing non 0-1 values for occupancy. I don't think that's a good idea for daylight schedules but if we want to use them then we should calculate the sum based on the same logic: `sum(0 if is_occ < 0.5 else 0 for is_occ in schedule)`

```python
return occ_pattern, sum(0 if is_occ < 0.5 for is_occ in schedule else 1)
```